### DISCUSSION TITLE
Is there a query.count() equivalent for async resultset ?

### DISCUSSION THREAD

**Discussion by ghevge** [sqlalchemy/sqlalchemy#6853] [Usage Questions]
Is there a query.count() equivalent for async resultset ?

Thanks

**Comment by CaselIT** [comment]
Hi,

Look at https://github.com/sqlalchemy/sqlalchemy/issues/6794

**Reply by CaselIT** [reply]
also of interest https://github.com/sqlalchemy/sqlalchemy/discussions/6334
